test/e2e/apimachinery/get_code_version.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
How about server_version.go instead of get_code_version.go as the test purpose.

Reply:
+1

Reply:
Fixed now. The original name was based on the `operational_id` from [APIsnoop](https://apisnoop.cncf.io)